### PR TITLE
Change tray icon

### LINKED ISSUE #2860: Change tray icon
It's still using lens-ide icons.

### PR DESCRIPTION
**Pull request by vshakirova**
Do we need the dark template? 

fixes: https://github.com/lensapp/lens/issues/2860 (mac os)
Screen Shot 2021-06-11 at 2 09 14 PM

### CODE REVIEW

**Comment by Nokel81** [comment]
Yes the dark template is for light theme

File: build/build_tray_icon.ts [hunk 1]
```diff
@@ -25,14 +25,14 @@ import fs from "fs-extra";
 
 export async function generateTrayIcon(
   {
-    outputFilename = "tray_icon", // e.g. output tray_icon_dark@2x.png
+    outputFilename = "trayIcon",
```
Comment (Nokel81) [inline build/build_tray_icon.ts:28]: Why were these changed?

Comment (vshakirova) [reply build/build_tray_icon.ts:28]: I've found the PR https://github.com/lensapp/lens/pull/1595 and I assumed that it would be useful to have `trayIconTempltate` file name after generating the icon. If I'm not mistaken for all OS (except mac os) we are using light icon for the tray. @aleksfront am I right? (sorry, I made a mistake and add the dark icon instead of light, I'll fix it).

**Review by aleksfront — APPROVED** [review]
Checked in macOS BigSur, macOS Catalina and Win 10. Looking good!

macOS BigSur
big sur tray icon

macOS Catalina

https://user-images.githubusercontent.com/9607060/124127702-bad52080-da84-11eb-9974-30b65fb2e8a4.mov

Windows
windows tray icon
